packages/ui/src/elements/Link/index.tsx

```diff
@@ -47,6 +55,12 @@ export const Link: React.FC<Parameters<typeof NextLink>[0]> = ({
           onClick(e)
         }
 
+        // We need a preventDefault here so that a clicked link doesn't trigger twice,
+        // once for default browser navigation and once for startRouteTransition
+        if (!disablePreventDefault) {
```

Comment:
Personally I would rather the prop be `preventDefault` and the check be `if (preventDefault !== false)`